pom.xml

```diff
@@ -185,7 +185,7 @@
             <dependency>
                 <groupId>org.jvnet.staxex</groupId>
                 <artifactId>stax-ex</artifactId>
-                <version>1.8.2</version>
+                <version>1.9.0-SNAPSHOT</version>
```

Comment:
Is there any version better than a snapshot?

Reply:
Unfortunately, there is not at the moment.